VRDR/DeathRecord.cs

```diff
@@ -7388,6 +7758,1877 @@ public string EmergingIssue20
+          component.Code = new CodeableConcept(CodeSystems.ComponentCode, "HispanicCodeForLiteral", "Hispanic Code For Literal", null);
+          component.Value = DictToCodeableConcept(value);
+          CodedRaceandEthnicityObs.Component.Add(component);
+      }
+  }
+
+  /// <summary>Hispanic Code For Literal  Helper</summary>
+  /// <value>Hispanic Code For Literal Helper.</value>
+  /// <example>
+  /// <para>// Setter:</para>
+  /// <para>ExampleDeathRecord.HispanicCodeForLiteralHelper = VRDR.ValueSets.RaceCode.African ;</para>
+  /// <para>// Getter:</para>
+  /// <para>Console.WriteLine($"Hispanic Code For Literal: {ExampleDeathRecord.HispanicCodeForLiteral}");</para>
+  /// </example>
+  [Property("Hispanic Code For Literal Helper", Property.Types.String, "Coded Content", "Hispanic Code For Literal.", true, IGURL.CodedRaceAndEthnicity, false, 34)]
+  [PropertyParam("code", "The code used to describe this concept.")]
+  [FHIRPath("Bundle.entry.resource.where($this is Observation).where(code.coding.code='codedraceandethnicity')", "")]
+  public string HispanicCodeForLiteralHelper
+  {
+      get
+      {
+          if (HispanicCodeForLiteral.ContainsKey("code"))
+          {
+              return HispanicCodeForLiteral["code"];
+          }
+          return null;
+      }
+      set
+      {
+          SetCodeValue("HispanicCodeForLiteral", value, VRDR.ValueSets.HispanicOrigin.Codes);
+      }
+  }
+
+
+
+
+
+        /// <summary>Entity Axis Cause Of Death</summary>
+        /// <value>Entity-axis codes</value>
+        /// <example>
+        /// <para>// Setter:</para>
+        /// <para>Tuple<string, string, string, string>[] eac = new Tuple<string, string, string, string>{Tuple.Create("value", "position", "line", "code")}</para>
+        /// <para>ExampleDeathRecord.EntityAxisCauseOfDeath = eac;</para>
+        /// <para>// Getter:</para>
+        /// <para>Console.WriteLine($"Emerging Issue Value: {ExampleDeathRecord.EntityAxisCauseOfDeath}");</para>
+        /// </example>
+        [Property("Entity Axis Cause of Death", Property.Types.Tuple4Arr, "Entity Axis Cause of Death", "", true, IGURL.EntityAxisCauseOfDeath, false, 50)]
+        [FHIRPath("Bundle.entry.resource.where($this is Observation).where(code.coding.code=80356-9)", "")]
+        public Tuple<string, string, string, string>[] EntityAxisCauseOfDeath
+        {
+            get
+            {
+                List<Tuple<string, string, string, string>> eac = new List<Tuple<string, string, string, string>>();
+                if (EntityAxisCauseOfDeathObs != null)
+                {
+                    foreach(Observation ob in EntityAxisCauseOfDeathObs)
+                    {
+                        CodeableConcept valueCC = (CodeableConcept)ob.Value;
+                        string value = valueCC.Coding[0].Code;
+                        Observation.ComponentComponent positionComp = ob.Component.Where(c => c.Code.Coding[0].Code=="position").FirstOrDefault();
```

Comment:
What happens if a position or lineNumber etc. is missing? Ideally we won't crash, and will just skip the field, perhaps while recording some type of validation error.

Reply:
Added in a check for null values, haven't recorded a validation error